### PR TITLE
[Console] Fix Console.Error to terminal (tty)

### PR DESCRIPTION
**Pull request by ludovic-henry**

### CODE REVIEW

**Comment by ludovic-henry** [comment]
@monojenkins merge

File: mcs/class/corlib/System/Console.cs [hunk 1]
```diff
@@ -141,45 +141,31 @@ static void SetupStreams (Encoding inputEncoding, Encoding outputEncoding)
...
+			} else
 #endif
-				stdout = TextWriter.Synchronized (stdout);
+			{
+				stdin = TextReader.Synchronized (new UnexceptionalStreamReader (OpenStandardInput (0), inputEncoding));
 
+#if MONODROID
+				if (LogcatTextWriter.IsRunningOnAndroid ()) {
+					stdout = TextWriter.Synchronized (new LogcatTextWriter ("mono-stdout", stdout));
+					stderr = TextWriter.Synchronized (new LogcatTextWriter ("mono-stderr", stderr));
```
Comment (akoeplinger) [inline mcs/class/corlib/System/Console.cs]: I think this is changing behavior here.
Before in the MONODROID case, stdout/stderr were initialized to a UnexceptionalStreamWriter before being passed to LogcatTextWriter.

Now they're not initialized at this point from what I can see.

Comment (ludovic-henry) [inline mcs/class/corlib/System/Console.cs]: You are right, let me correct that. I also removed the comment to automerge.

**Comment by ludovic-henry** [comment]
@akoeplinger I updated the code, would you mind checking again? Thank you

**Comment by akoeplinger** [comment]
LGTM now. Please include `Fixes https://bugzilla.xamarin.com/show_bug.cgi?id=36988` in the first commit though so we have cross-reference.

**Comment by ludovic-henry** [comment]
@monojenkins merge

**Pull request merged**